.github/workflows/check-rebase.yml

```diff
@@ -0,0 +1,34 @@
+---
+name: "Check Rebase"
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    branches:
+      - master
+      - v1.*
+
+jobs:
+  checkRebase:
+    runs-on: ubuntu-latest
+    env:
+      PR_NUMBER: "${{ github.workspace }}"
+    steps:
+      - name: Checkout Head
+        uses: actions/checkout@v2
+        with:
+          repository: ${{github.event.pull_request.head.repo.full_name}}
+          fetch-depth: 0
+      - name: Checkout Base
+        uses: actions/checkout@v2
+        with:
+          repository: ${{github.event.pull_request.base.repo.full_name}}
+          fetch-depth: 0
+      - run: |
```

Comment:
define the head and base SHA as environment variable to make the code more readable